doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -5281,6 +5331,80 @@ Note the optional "limit" parameter. Without it, the API will attempt to populat
 
 By default, the admin API calls are blocked and can only be called from localhost. See more details in :ref:`:BlockedApiEndpoints <:BlockedApiEndpoints>` and :ref:`:BlockedApiPolicy <:BlockedApiPolicy>` settings in :doc:`/installation/config`.
 
+Get File External Tool URL
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This API call generates a URL for accessing an external tool (see :doc:`/installation/external-tools`) that operates at the file level. The URL includes necessary authentication tokens and parameters based on the user's permissions and the tool's configuration.
+
+Authentication is required for draft, restricted, embargoed, or expired (retention period) files, the user must have appropriate permissions.
+
+.. code-block:: bash
+
+  export API_TOKEN=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+  export SERVER_URL=https://demo.dataverse.org
+  export FILE_ID=42
+  export TOOL_ID=3
+
+  curl -H "X-Dataverse-key:$API_TOKEN" -X POST "$SERVER_URL/api/files/$FILE_ID/externalTool/$TOOL_ID/toolUrl" \
+  -H "Content-Type: application/json" \
+  -d '{"preview": false, "locale": "en"}'
+
+The fully expanded example above (without environment variables) looks like this:
+
+.. code-block:: bash
+
+  curl -H "X-Dataverse-key:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx" -X POST "https://demo.dataverse.org/api/files/42/externalTool/3/toolUrl" \
+  -H "Content-Type: application/json" \
+  -d '{"preview": false, "locale": "en"}'
+
+The JSON request body accepts the following optional parameters:
+- ``preview``: boolean flag to indicate if the tool should run in preview mode (default: false)
+- ``locale``: string specifying the locale for internationalization
+
+The response includes:
+- ``toolUrl``: the URL to access the external tool
+- ``toolName``: the display name of the external tool
+- ``fileId``: the ID of the file
+- ``preview``: whether the URL is for preview mode
```

Comment:
The HTML bullets are not right: https://dataverse-guide--11760.org.readthedocs.build/en/11760/api/native-api.html#get-file-external-tool-url